runtime/doc/options.txt

```diff
@@ -2827,10 +2827,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 		:s///g		  subst. one	  subst. all
 		:s///gg		  subst. all	  subst. one
 
-	DEPRECATED: Setting this option may break plugins that are not aware
-	of this option.  Also, many users get confused that adding the /g flag
-	has the opposite effect of that it normally does.
```

Comment:
Suggestion: replace with
```
NOTE: Setting this option may break plugins that rely on the default behavior of the `/g` flag.
```

Reply:
(Also, this documentation is generated from `options.lua`, so needs to be changed there as well.)

Reply:
Might also want to make a PR to vim with the same wording; if you do it first, we may not have to edit it twice if they decide on a different wording.